entries/repo-website-overview.puml

```diff
@@ -0,0 +1,80 @@
+@startuml Repos-Sites
+
+  package .github as GIT {
+      diamond dia1
+  }
+
+  package data as DAT {
+      diamond dia2
+  }
+
+  package theme as THE {
+      diamond dia3
+  }
+
+  package community as COM {
+      object "Community.Seedcase-Project.org" as CSP {
+        Project blocks
+        Roadmap
+        Guides
+        Posts
+        Outreach
+        Code of Conduct
+      }
+  }
+
+  package decisions as DEC {
+      object "Decisions.Seedcase-Project.org" as DeCP {
+        Posts
+      }
+  }
+
+  package design as DES {
+      object "Design.Seedcase-Project.org" as DeSP {
+        Welcome
+        Software architecture documentation
+      }
+  }
+
+  package team as TEA {
+      object "Team.Seedcase-Project.org" as TSP {
+        Onboarding
+        Entries
+        User Tests
+        Events
+      }
+  }
+
+  package "Seedcase Sprout" as SPROUT {
+      object "localhost" as SSP {
+        App for upload
+        and storing data 
+      }
+  }
+
+  package seedcase {
+      object "About Seedcase" as ABOUT {
+        history
+        team members 
+      }
+
+      object "Learn" as LEA {
+        Learning Shorts
+      }
```

Comment:
Will/should `learn` be moved to it's own repo? It seems unintuitive that this is the only one that doesn't have its own repo?

Reply:
I'll make a discussion issue for it.